packages/wallets/src/components/WalletsActionScreen/index.ts

```diff
@@ -1 +1 @@
-export { default as WalletResetBalance } from './WalletsActionScreen';
+export { default as WalletsActionScreen } from './WalletsActionScreen';
```

Comment:
The component is `<WalletActionScreen />` however the filename and the `WalletsActionScreen`

Reply:
You can fix it in your PR man. The coveralls runner cant take another commit